src/main/java/com/shopify/model/MetafieldValueType.java

```diff
@@ -12,7 +15,11 @@ private MetafieldValueType(final String value) {
 	}
 
 	public static MetafieldValueType toEnum(final String value) {
```

Comment:
can you change the variable name of the argument here to type to remove the ambiguity.